lib/grandpa/grandpa.go

```diff
@@ -44,6 +44,8 @@ var (
 	// TODO: make this configurable; currently 1s is same as substrate; total round length is interval * 2 (#1869)
 	interval = time.Second
 	logger   = log.New("pkg", "grandpa")
+
+	errUnsupportedSubround = errors.New("unsupported subround")
```

Comment:
We should export it since we want to wrap the 'bottom' error, unless the error is logged out and thrown out in the grandpa package.

Reply:
Done!